dwellingplace/models/_utils.py

```diff
@@ -32,8 +32,8 @@ def parse_xlsx_into_dicts(xl):
                     metric_dict[col_name] = sheet.cell(row, col).value
             # special conversions
             try:
-                parts = xlrd.xldate_as_tuple(metric_dict['Date'], xl.datemode)
-                metric_dict['Date'] = datetime(*parts)
+                year, month, *_ = xlrd.xldate_as_tuple(metric_dict['Date'], xl.datemode)
+                metric_dict['Date'] = datetime(year, month, 1, 0, 0, 0)
```

Comment:
I think we should adjust the date to 1st of the month in the import, rather than export, so that database and JSON export has consistent days too.